client/js/views/Rules.vue

```diff
@@ -380,6 +398,24 @@ export default {
       }
       return 'Sign Up to Play Online';
     },
+    videoWidth() {
+      switch (this.$vuetify.breakpoint.name) {
+        case 'xs':
+        case 'sm':
+          return 300;
+        case 'md':
+          return 560;
+        case 'lg':
+          return 800;
+        case 'xl':
+          return 1000;
+        default:
+          return 560;
+      }
+    },
+    videoHeight() {
+      return (this.videoWidth * 315) / 560;
+    },
```

Comment:
Would it be better to set a padding-top percent and make the video responsive in nature?

Reply:
https://codepen.io/seriouslysean/pen/RwMwmYq

Reply:
absolutely, this is better. Done